python-sdk/examples/opencv/opencv.py

```diff
@@ -72,12 +72,17 @@ def main():
         # Add some images, recursively inserting content from the images
         # directory. Alternatively, you could use `client.put_file_url` or
         # `client_put_file_bytes`.
-        source = None  # TODO: Add correct path
+        source = relpath("images")
         client.pfs.put_files(commit=commit, source=source, path="/")
 
     # Wait for the commit (and its downstream commits) to finish
     commit.wait_set()
 
+    job = pps.Job(pipeline=pps.Pipeline(name="montage"), id=commit.id)
+    if client.pps.inspect_job(job=job).state != pps.JobState.JOB_SUCCESS:
+        print("Montage job failed, aborting. Check the pipeline logs for more details.")
```

Comment:
Since we just made the call to retrieve the JobInfo, it would be nice to print out the exact pachctl command used to get the logs.